benchmark/realnest/3_list_transform_plus_list_aggregate.benchmark

```diff
@@ -0,0 +1,24 @@
+# name: benchmark/realnest/3_list_transform_plus_list_aggregate.benchmark
+# description: select average from transformed list and group by, having
+# group: [realnest]
+
+name list_transform plus list_aggregate
+group real_nest
+
+require json
+
+require httpfs
+
+load benchmark/realnest/load.sql
+
+run
+SELECT PV.npvs as npvs, 
+  avg(list_aggregate(list_transform(Jet, x -> x.pt), 'avg')) as avg_pt,
```

Comment:
If I'm not mistaken, the dataset has many more columns with a list of structs.
To really stress test list_transform + list_aggregate you could also caluclate some aggregates for the following columns

`Muon`, `Electron`, `Tau`, and `Photon`

Reply:
Added all of them to the `04_list_transform_plus_list_aggregate` benchmark